server/engine/src/main/java/org/eclipse/lsp/cobol/domain/modules/ServiceModule.java

```diff
@@ -73,6 +73,13 @@ protected void configure() {
     bind(DisposableLSPStateService.class).to(CobolLSPServerStateService.class);
     bind(LanguageEngineFacade.class).to(CobolLanguageEngineFacade.class);
     bind(CopybookService.class).to(CopybookServiceImpl.class);
+    bind(PredefinedCopybookStore.class).to(PredefinedCopybookStoreImpl.class);
+    // TODO: configure below commented binding based on some flag
+    bind(ResolveCopybookUri.class).to(NonCacheResolveCopybookUri.class);
+    bind(ResolveFileContent.class).to(ClientBasedFileContent.class);
+    //    bind(ResolveCopybookUri.class).to(NonCacheResolveCopybookUri.class);
+    //    bind(ResolveFileContent.class).to(ClientBasedFileContent.class);
```

Comment:
Maybe we don't care anymore?
```suggestion
    bind(ResolveCopybookUri.class).to(NonCacheResolveCopybookUri.class);
    bind(ResolveFileContent.class).to(ClientBasedFileContent.class);
```